go.mod

```diff
@@ -5,12 +5,12 @@ go 1.18
 require (
 	github.com/golang/protobuf v1.5.3
```

Comment:
```
github.com/golang/protobuf v1.5.3
```
^ this keep creeping in, perhaps because of the go_opt flags pointing to it. i wonder if upgrading those package refs to avoid this repo would help?

Reply:
this is already fixed in the 3rd commit of this PR, please check.